org.lflang/src/org/lflang/generator/ReactionInstance.java

```diff
@@ -448,6 +448,57 @@ public String toString() {
         return getName() + " of " + parent.getFullName();
     }
 
+    /**
+     * Determine logical execution time for each reaction during compile
+     * time based on immediate downstream logical delays (after delays and actions)
+     * and label each reaction with the minimum of all such delays.
+     */
+    public TimeValue getLogicalExecutionTime() {
```

Comment:
I think this should be renamed to `assignLogicalExecutionTime()` as it assigns LET rather than getting it.

Reply:
OK. I renamed the function name to `assignLogicalExecutionTime()`. https://github.com/lf-lang/lingua-franca/pull/1292/commits/3ca6084fa42d61554184d13bf47e9d754a153eb7